dev-tools/ci-integration/step2-extra_args.sh

```diff
@@ -0,0 +1 @@
+EXTRA_ARGS=('--path-mode=intersection' '--' "${CHANGED_FILES[@]}")
```

Comment:
I have split original first step into 2 steps, so it will be easier to adjust #2925 to testing script